native-oci-generative-ai/pom.xml

```
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.oracle.labs</groupId>
+  <artifactId>generative-ai</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <name>Oracle Cloud Infrastructure - Generative AI</name>
```

Should this include the word "Demo" somewhere?
```suggestion
  <name>Oracle Cloud Infrastructure - Generative AI Demo</name>
```